src/components/tables/TableActionsDropdown.tsx

```diff
@@ -0,0 +1,64 @@
+import { DeleteOutlined, MoreOutlined } from '@ant-design/icons';
+import { Button, Dropdown, MenuProps } from 'antd';
+import { useMemo, useState } from 'react';
+import { DeleteTableModal } from '../modals/DeleteTableModal';
+
+interface TableActionDropdownProps {
+  catalog: string;
+  schema: string;
+  table: string;
+  tableFullName: string;
+}
+
+enum TableActionsEnum {
+  Delete,
+  Rename
+}
+
+
+export default function TableActionsDropdown({
+  catalog,
+  schema,
+  table,
+  tableFullName,
```

Comment:
nit: is a bit duplicated.
`tableFullName` always equal to `${catalog}.${schema}.${table}`